### PR TITLE
Worldedit Wand

### PR DESCRIPTION
**Pull request by pickardjoe**
Added a WorldEdit wand item that can be used to select areas with WorldEdit. Inspired by a mod from another voxel-based game.

### CODE REVIEW

**Comment by sfan5** [comment]
please fix:
* your code style:
* &nbsp; Lua is not C, don't add semicolons
* &nbsp; `if` doesn't need so many
* a texture being missing

**Comment by pickardjoe** [comment]
@sfan5 I've implemented these changes.

**Comment by sfan5** [comment]
47712844a3f2551aedfa9e39cc3a8cc06e01993e

File: worldedit/wand.lua [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    tool_capabilities = {
+        full_punch_interval = 1.0,
+        max_drop_level=0,
+        groupcaps={
+            fleshy={times={[2]=0.80, [3]=0.40}, maxwear=0.05, maxlevel=1},
+            snappy={times={[2]=0.80, [3]=0.40}, maxwear=0.05, maxlevel=1},
+            choppy={times={[3]=0.90}, maxwear=0.05, maxlevel=0}
+        }
+    },
+    node_placement_prediction = nil,
```
Comment (ShadowNinja) [inline worldedit/wand.lua]: I'm pretty sure these are set by default by register_tool.

**Pull request closed without merging**